config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
> AFAIK they are probably the most widely used configuration, albeit they have the problems we've discussed meaning we want to steer people away from it to pod discovery, but with enhanced labelling.

You're thinking of services, not service endpoints.

Service endpoints are only useful for blackbox monitoring.
Services are the widely used configuration, which currently have the healthcheck problem.

What I don't want is a user seeing this and thinking it's okay to use service endpoints for whitebox monitoring.